### PR TITLE
Kind presubmits

### PR DESCRIPTION
**Pull request by BenTheElder**
- move kubernetes kind presubmits to owning sig (sig-testing) and a seperate file vs kubeadm periodics
- change the kubernetes master branch presubmit to use our shiny new latest tested binaries instead of cloning & building kind, also stop pointlessly cloning the kubeadm repo (some legacy config?)

/cc @amwat

### CODE REVIEW

File: config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+    decoration_config:
+      timeout: 40m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: kubeadm
```
Comment (aojea) [inline config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml]: do we still need kubeadm references?

File: config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    - org: kubernetes-sigs
+      repo: kind
+      base_ref: master
+      path_alias: sigs.k8s.io/kind
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191001-33df843-1.14
+        command:
+        - runner.sh
+        - kubetest
```
Comment (aojea) [inline config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml]: kubetest is needed?

File: config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml]: i'm leaving the old branches alone for now

File: config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml (hunk 2 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml]: it is used in the old branches

**Comment by BenTheElder** [comment]
only the newest branch was changed, the rest are just moved.

**Comment by aojea** [comment]
LGTM

**Comment by neolit123** [comment]
LGTM

**Pull request merged**